docs/index.rst

```diff
@@ -152,9 +152,10 @@ then the *hookimpl* should be marked with the ``"optionalhook"`` option:
 
 Call time order
 ^^^^^^^^^^^^^^^
-A *hookimpl* can influence its call-time invocation position.
-If marked with a ``"tryfirst"`` or ``"trylast"`` option it will be
-executed *first* or *last* respectively in the hook call loop:
+By default hooks are :ref:`called <calling>` in LIFO registered order, however,
+a *hookimpl* can influence its call-time invocation position using special
+attributes. If marked with a ``"tryfirst"`` or ``"trylast"`` option it
+will be executed *first* or *last* respectively in the hook call loop:
```

Comment:
What happens when more than one hookimpl are marked as `tryfirst` or `trylast`? I believe then LIFO registered order for them applies, but perhaps this should be explicitly mentioned.

Reply:
Ok so something like:

> Note that `tryfirst` and `trylast` hooks are still invoked in LIFO order within each category

Reply:
Great 👍

Reply:
done